simple/markupsafe/index.html

```diff
@@ -11,5 +11,6 @@ <h1>Links for markupsafe</h1>
     <a href="/localwheels/MarkupSafe-1.0-cp35-cp35m-linux_x86_64.whl">MarkupSafe-1.0-cp35-cp35m-linux_x86_64.whl</a><br/>
     <a href="/localwheels/MarkupSafe-1.1.0.tar.gz">MarkupSafe-1.1.0.tar.gz</a><br/>
     <a href="/localwheels/MarkupSafe-1.1.0-cp35-cp35m-linux_x86_64.whl">MarkupSafe-1.0-cp35-cp35m-linux_x86_64.whl</a><br/>
+    <a href="/localwheels/MarkupSafe-1.1.0-cp37-cp37m-linux_x86_64.whl">MarkupSafe-1.0-cp37-cp37m-linux_x86_64.whl</a><br/>
```

Comment:
version here in the href is 1.1.0, wheel has version 1.0 - is this a bug in one of our scripts?

Reply:
Yup, it was typo from an year ago.